UnitsNet/GeneratedCode/Quantities/Mass.NetFramework.g.cs

```diff
@@ -64,7 +64,32 @@ public partial struct Mass : IQuantity<MassUnit>, IEquatable<Mass>, IComparable,
         static Mass()
         {
             BaseDimensions = new BaseDimensions(0, 1, 0, 0, 0, 0, 0);
-            Info = new QuantityInfo<MassUnit>(QuantityType.Mass, Units, BaseUnit, Zero, BaseDimensions);
+
+            Info = new QuantityInfo<MassUnit>(QuantityType.Mass, new UnitInfo<MassUnit>[] {
+                new UnitInfo<MassUnit>(MassUnit.Centigram, new BaseUnits(LengthUnit.Undefined, MassUnit.Undefined, DurationUnit.Undefined, ElectricCurrentUnit.Undefined, TemperatureUnit.Undefined, AmountOfSubstanceUnit.Undefined, LuminousIntensityUnit.Undefined)),
```

Comment:
It's quite verbose right now, since the majority is assigning a single value we can later use default values to omit most of them.